meeting-notes/2019/Q1/2019-02-27--gui-and-in-web-browsers-weekly.md

```diff
@@ -0,0 +1,218 @@
+# GUI & In Web Browsers Weekly Sync 2019-02-27
+
+-   Moderator: @olizilla
+-   Notetaker: @lidel
+-   Attendees:
+    -   _@jimpick_
+    -   _@terichadbourne_
+    -   _@andrew_
+    -   _@olizilla_
+    -   _@hugomrdias_
+    -   _@fsdiogo_
+    -   _@cwaring_
+-   Recording: <https://youtu.be/hGkVST8H-Q8> || [/ipfs/bafybeieiddxcwyve3pfxz4su2trokzr3kw6e3ojbwj32iu2g5pksk6camy](https://ipfs.io/ipfs/bafybeieiddxcwyve3pfxz4su2trokzr3kw6e3ojbwj32iu2g5pksk6camy) 
+
+ 
+
+
+**Preflight checklist**
+
+-   Call for additional agenda items!
+-   Ensure that there is a notetaker!
+-   Start recording! :^)
+
+  
+
+
+**Agenda**
+
+_General discussions, decisions, etc._
+
+-   Start with Team Updates and Demos
+-   Web UI help system
+-   Analytics
+
+**Team Updates**
+
+  
+
+
+@hacdias
+
+-   Done:
+    -   dnslink-cloudflare: a package to update the dnslink DNS record on Cloudflare enabled websites ([Code](https://github.com/ipfs-shipyard/dnslink-cloudflare))
+    -   (ipfs-desktop) Released 0.7.0 rc2 ([release](https://github.com/ipfs-shipyard/ipfs-desktop/releases/tag/v0.7.0-rc.2))
+        -   2 friends and I updated successfully on Windows without any issues.
+    -   (ipfs-desktop) Check for updates every hour and add option to manually check for updates in the context menu ([PR](https://github.com/ipfs-shipyard/ipfs-desktop/pull/841))
+    -   (ipfs-desktop) Update Menubar’s language according to Web UI’s ([PR](https://github.com/ipfs-shipyard/ipfs-desktop/pull/847))
+    -   (ipfs-webui) **NEEDS REVIEW** Add Connection ([PR](https://github.com/ipfs-shipyard/ipfs-webui/pull/969))
+    -   (ipfs-webui) Random fixes ([PR 972](https://github.com/ipfs-shipyard/ipfs-webui/pull/972))
+
+  
+
+
+@lidel
+
+-   Done:
+    -   (is-ipfs) PR to add multiaddr checks: <https://github.com/ipfs/is-ipfs/pull/27> 
+    -   (ipfs-companion)
+        -   Per site redirect opt-out ready for review: (<https://github.com/ipfs-shipyard/ipfs-companion/pull/687>)
+        -   (small win) hapi v18 (HTTP server used by js-ipfs) works in Brave over chrome.sockets: <https://github.com/ipfs-shipyard/ipfs-companion/issues/664#issuecomment-467854030> 
+    -   (i18n) deep dive into [Hugo's Multilingual Mode](https://gohugo.io/content-management/multilingual/) 
+    -   (js-mafmt) fix support Buffer input: <https://github.com/multiformats/js-mafmt/pull/39> 
+-   Blocked:
+    -   <https://github.com/ipfs/is-ipfs/pull/27> by:
+        -   <https://github.com/ipfs/aegir/issues/336> 
+        -   <https://github.com/multiformats/js-mafmt/pull/39> 
+-   Next:
+    -   Merge unblocked/reviewed PRs & release new Beta of ipfs-companion
+
+  
+
+
+@olizilla
+
+-   Done:
+    -   Released ipfs-dns-deploy for deploying websites to IPFS via circle
+        -   https&#x3A;//github.com/ipfs-shipyard/ipfs-dns-deploy
+    -   Released webui 2.4
+    -   CI for ipfs website <https://github.com/ipfs/website/pull/294> 
```

Comment:
@olizilla is there any concern that we're using Travis.com for js-ipfs and circle for our websites?